app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -230,6 +358,34 @@ object ConsoleCli {
     }
 
     val requestParam: RequestParam = config.command match {
+      // DLCs
+      case CreateDLCOffer(amount,
+                          oracleInfo,
+                          contractInfo,
+                          feeRateOpt,
+                          locktime,
+                          refundLT,
+                          escaped) =>
+        RequestParam(
+          "createdlcoffer",
+          Seq(
+            up.writeJs(amount),
+            up.writeJs(oracleInfo),
+            up.writeJs(contractInfo),
+            up.writeJs(feeRateOpt),
+            up.writeJs(locktime),
+            up.writeJs(refundLT),
+            up.writeJs(escaped)
+          )
```

Comment:
This gives
```
could not find implicit value for evidence parameter of type upickle.default.Writer[T]
```
I think that is because it gives us a `Vector[Any]` and we would then need a js Writer for `Any`